app/models/models.rb

```diff
@@ -0,0 +1,5 @@
+class Event < ActiveRecord::Base
+end
+
+class Style <ActiveRecord::Base
+end
```

Comment:
While it works to put models in the same file, please separate them out into their own files.